contracts/RealTokenizado.sol

```diff
@@ -18,13 +22,31 @@ contract RealTokenizado is RealDigital {
         uint256 _cnpj8,
         address _reserve
     ) RealDigital(_name, _symbol, _authority, _admin) {
+        addressDiscovery = _ad;
         participant = _participant;
         cnpj8 = _cnpj8;
         reserve = _reserve;
+
+        _setRoleAdmin(ACCESS_ROLE, ACCESS_FREEZER_MOVER_ADMIN);
+        _setRoleAdmin(FREEZER_ROLE, ACCESS_FREEZER_MOVER_ADMIN);
+        _setRoleAdmin(MOVER_ROLE, ACCESS_FREEZER_MOVER_ADMIN);
+
+        _grantRole(ACCESS_FREEZER_MOVER_ADMIN, _admin);
+        RealDigitalDefaultAccount rdda = getRealDigitalDefaultAccount();
+        _grantRole(ACCESS_FREEZER_MOVER_ADMIN, address(rdda));
     }
 
     function updateReserve(address _newReserve) public {
-        require(_msgSender() == authority, "RealTokenizado: Only authority can update reserve");
```

Comment:
it's not clear from the docs what the access rules here are.
I assumed a participant will manage their own reserve through the defaultAccount, but I'm not 100% sure here.